pages/installation/os/sles.rst

```diff
@@ -0,0 +1,122 @@
+*******************
+SLES installation
+*******************
+
+This guide describes the fastest way to install Graylog on SLES 12 SP3. All links and packages are present at the time of writing but might need to be updated later on.
+
+.. warning:: This setup should not be done on publicly exposed servers. This guide **does not cover** security settings!
+
+
+Prerequisites
+-------------
+
+The following Patterns are use as a minimal server setup::
+
+  - Base System
+  - Minimal System (Appliances)
+  - Yast configuration packages
+
+.. warning:: This Guide makes the assumption that the firewall is disabled and communication is possible to the outside world.
+
+Taking a minimal server setup as base will need this additional packages::
+
+  $ sudo zypper install java-1_8_0-openjdk
+
+
+MongoDB
+-------
+
+Installing MongoDB on SLES should follow `the tutorial for SLES <https://docs.mongodb.com/v3.4/tutorial/install-mongodb-on-suse/>`_ from the MongoDB documentation. Add the GPG Key and the Repository before you install MongoDB::
```

Comment:
Key &rarr; key
Repository &rarr; repository
"before you install MongoDB" &rarr; "before installing MongoDB"